manifests/09-dedicated-admins.SubjectPermission.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: dedicated-admins
+  namespace: openshift-rbac-permissions
+spec:
+  subjectKind: Group
+  subjectName: dedicated-admins
+  clusterPermissions:
+    - dedicated-admins-cluster
+  permissions:
+    - clusterRoleName: dedicated-admins-project
+      namespacesAllowed: "*"
+      namespacesDenied: "default,kube.*,openshift.*"
```

Comment:
in regex form? "(^kube-.*|^openshift.*|^ops-health-monitoring$|^management-infra$|^default$|^logging$|^sre-app-check$)"

Reply:
@sam-nguyen7 Are you saying this is what it should be?

Reply:
yes I believe so